Makefile

```diff
@@ -0,0 +1,13 @@
+# Package configuration
+PROJECT = gemini
+
+# Including ci Makefile
+MAKEFILE = Makefile.main
+CI_REPOSITORY = https://github.com/src-d/ci.git
+CI_FOLDER = .ci
```

Comment:
Why are we using this names and import rule? It seems to be compared from somewhere but [the current recommendations](https://github.com/src-d/ci/blob/master/examples/ci/Makefile)

Reply:
https://github.com/bblfsh/dashboard/blob/master/Makefile#L7
don't see much difference between it and the example. they do the same thing.